airflow/utils/db.py

```diff
@@ -68,38 +68,44 @@
 log = logging.getLogger(__name__)
 
 REVISION_HEADS_MAP = {
+    "1.5.0": "338e90f54d61",
+    "1.5.2": "52d714495f0",
+    "1.6.0": "40e67319e3a9",
+    "1.6.2": "4446e08588",
+    "1.7.0": "1968acfc09e3",
+    "1.7.1": "2e82aab8ef20",
+    "1.8.2": "d2ae31099d61",
+    "1.10.0": "9635ae0956e7",
+    "1.10.2": "41f5f12752f8",
+    "1.10.3": "a56c9515abdc",
+    "1.10.4": "004c1210f153",
+    "1.10.7": "fe461863935f",
+    "1.10.5": "74effc47d867",
+    "1.10.8": "7939bcff74ba",
+    "1.10.10": "952da73b5eff",
+    "1.10.11": "a66efa278eea",
+    "1.10.12": "da3f683c3a5a",
+    "1.10.13": "03afc6b6f902",
     "2.0.0": "e959f08ac86c",
     "2.0.1": "82b7c48c147f",
     "2.0.2": "2e42bb497a22",
     "2.1.0": "a13f7613ad25",
-    "2.1.1": "a13f7613ad25",
-    "2.1.2": "a13f7613ad25",
     "2.1.3": "97cdd93827b8",
     "2.1.4": "ccde3e26fe78",
     "2.2.0": "7b2661a43ba3",
-    "2.2.1": "7b2661a43ba3",
-    "2.2.2": "7b2661a43ba3",
     "2.2.3": "be2bfac3da23",
     "2.2.4": "587bdf053233",
-    "2.2.5": "587bdf053233",
     "2.3.0": "b1b348e02d07",
     "2.3.1": "1de7bc13c950",
     "2.3.2": "3c94c427fdf6",
     "2.3.3": "f5fcbda3e651",
-    "2.3.4": "f5fcbda3e651",
     "2.4.0": "ecb43d2a1842",
-    "2.4.1": "ecb43d2a1842",
     "2.4.2": "b0d31815b5a6",
     "2.4.3": "e07f49787c9d",
     "2.5.0": "290244fb8b83",
-    "2.5.1": "290244fb8b83",
-    "2.5.2": "290244fb8b83",
-    "2.5.3": "290244fb8b83",
     "2.6.0": "98ae134e6fff",
```

Comment:
I think we miss some revisions?

Reply:
Yes, I account for them in the db_command.py. The ones missing are those that don't have migration. Previously, we fill it with the value for the previous version that had migration but now, I search recursively in db_command.py to find the version with migration if the given version doesn't have migration

Reply:
I see. Yeah. Makes sense. and NO. REVISION_HEADS_MAP is NOT mentioned as part of Public Airflow interface https://airflow.apache.org/docs/apache-airflow/stable/public-airflow-interface.html - so we are completely safe to change it. If someone uses it, we can send them to the page (I am really happy I started the PR about this one - I've already fended-off a number of claims that things should not change :D ) .

Reply:
And yes - feel free to add _ in front as well - this is even stronger sign to "you should not rely on it"